x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/__tests__/monitor_list_pagination.test.tsx

```diff
@@ -1,126 +0,0 @@
-/*
```

Comment:
This component is deleted?

Reply:
I moved these tests [here](https://github.com/elastic/kibana/pull/62853/files/f1ad9d405062a664aaf036732d0e6e99b72ab36b#diff-e56af0c7649951b70833c74e197feef0R165) because they're running tests on `MonitorListComponent`.